backend/control/control.go

```diff
@@ -209,3 +218,24 @@ func GetAwsProfiles() (AwsProfilesResponse, error) {
 	}
 	return apr, nil
 }
+
+type VulnRuleInfo struct {
+	CveIdentifier string  `json:"id,omitempty"`
+	Name          string  `json:"name,omitempty"`
+	Description   string  `json:"description,omitempty"`
+	CvssScore     float64 `json:"cvss_score,omitempty"`
+	YamlTemplate  string  `json:"yaml_template,omitempty"`
```

Comment:
I think only @varunjain99 @Tushar200018 only Yaml Template should be omitempty right?